lib/DDG/Spice/NewYorkTimes.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::NewYorkTimes;
```

Comment:
Personally, I think this package name should be a little more specific, because it's not clear if this is for news for book reviews, or something else. They have several APIs that could be used for a Spice.

We could also namespace it if we plan to use others: ie. `DDG::Spice::NewYorkTime::BookReview` or `DDG::Spice::NytBookReview`